### PR TITLE
Batch Feeless Verification Transactions

### LINKED ISSUE #6214: Komenci transactions should be batched when possible
### Expected Behavior
Transactions are batched when submitted to Komenci.

### Current Behavior
Transactions are not batched.

Part of epic #6102

### LINKED ISSUE #6215: Verification should show error banner when feeless verification encounters an error
### Expected Behavior
Failure banner is shown when error occurs during feeless flow

### Current Behavior
Failure banner is not shown when error occurs during feeless flow

### PR DESCRIPTION
**Pull request by tarikbellamine**
### Description
This PR builds off of the back-end work in [this PR](https://github.com/celo-org/komenci/pull/142) and batches transactions to the extent possible when submitting to Komenci. At this point, it is primarily happening in two places:
- the `approve` + `request` transactions that occur for each attestation request
- the `withdraw` + `transfer` transactions that occur during escrow redemption

### Other changes
N/A

### Tested
Yes

### Related issues
- Part of #6102
- Fixes #6214

### Backwards compatibility
Yes

### CODE REVIEW

File: packages/mobile/src/identity/reducer.ts
Comment (gnardini) [inline packages/mobile/src/identity/reducer.ts]: Why is the `> 0` not needed anymore? Are those states simply not possible anymore? Even if they're not I don't see an issue with leaving it just in case, but I might be missing something.

**Review by gnardini — APPROVED** [review]
Just a question, but looks good! :)

**Review by jeanregisser — APPROVED** [review]
Looks great! 👍

File: packages/mobile/src/identity/reducer.ts
Comment (tarikbellamine) [reply packages/mobile/src/identity/reducer.ts]: The issue was that we were using `feelessVerificationState.isActive` as a way to know whether to check for the feeless verification error or the non-feeless verification error. When we had `status > 0` that would set `isActive = false` whenever there was an error, which would cause the UI to check for non-feeless errors.

This is messy logic and mostly just a bandage that will hopefully be simplified with the verification refactor work.

**Comment by tarikbellamine** [comment]
We need PR #6178 to be completed in order to properly test transaction batching

**Comment by i1skn** [comment]
@tarikbellamine is this blocked by something? If yes, how can we help to unblock it?

File: packages/mobile/src/identity/feelessVerification.ts
Comment (bowd) [inline packages/mobile/src/identity/feelessVerification.ts]: All this can be reverted back to the call to:
```
komenciKit.requestAttestations(...)
```
And the approve step can be removed because the requestAttestations method will take care of crafting the approve transaction as well and pass that to the API.

**Review by bowd — CHANGES_REQUESTED** [review]

File: packages/mobile/src/escrow/saga.ts
Comment (bowd) [inline packages/mobile/src/escrow/saga.ts]: I figured it out you don't need to use the sign and execute here.
You just need to pass in the `withdrawAndTransferTx` straight to `komenciKit.submitMetaTransaction` and that takes care of the signing and stuff. The way you did it now results in a double wrapped meta transaction that doesn't pass the filter.

**Review by bowd — APPROVED** [review]

**Comment by Lss-Ankit** [comment]
Hi @tarikbellamine  I have verified this issue using latest Android Internal Build V 1.10.1(1004294336) & Test Flight Build V1.10.1(46) & observed following observations.

- [ ] User get "Verification Error" popup with skip for now button when user try to paste last code by tapping on "Resend Last Message" button for Android Internal Build V 1.10.1(1004294336) 
 **Attachment:**
[VerificationErrorWhilePestingLastCode.mp4](https://drive.google.com/file/d/1AaWRhpMYTfr-4Z-r2uTI12aQAEZyoagu/view?usp=sharing)
[AndroidVerificationErrorLog.txt](https://drive.google.com/file/d/1vfElxfM7OcymmQwKeo-t6nmUg6ow7EbZ/view?usp=sharing)
**Frequency:** 100%
- [x] For Test Flight Build V1.10.1(46)  build user able to proceed to verification & Home screen without any error message. 
**Verified on Devices:** Samsung Galaxy A5(7.0), iPhone 12(14.0)

Let me know if anything need to test.
Thanks..!!

**Pull request merged**
